e2e/support/component-webpack.config.js

```diff
@@ -81,16 +81,55 @@ module.exports = {
 };
 
 function resolveEmbeddingSdkPackage() {
-  let isEmbeddingSdkPackageInstalled = false;
-
   try {
-    const packagePath = require.resolve("@metabase/embedding-sdk-react");
-    if (packagePath.includes("node_modules")) {
-      isEmbeddingSdkPackageInstalled = true;
+    const nodeModulesPath = path.resolve(__dirname, "../../node_modules");
+
+    const sdkInNodeModulesPath = path.resolve(
+      nodeModulesPath,
+      SDK_PACKAGE_NAME,
+    );
+
+    // Try to resolve the package from the ancestor node_modules folder
+    const hasEmbeddingSdkInNodeModules = fs.existsSync(sdkInNodeModulesPath);
+
+    if (hasEmbeddingSdkInNodeModules) {
+      return {
+        isEmbeddingSdkPackageInstalled: true,
+        embeddingSdkPath: sdkInNodeModulesPath,
+      };
     }
-  } catch (err) {}
+
+    // Try to resolve the package with `require.resolve`
+    const requirePackagePath = require.resolve(SDK_PACKAGE_NAME);
+
+    if (requirePackagePath.includes("node_modules")) {
+      return {
+        isEmbeddingSdkPackageInstalled: true,
+        embeddingSdkPath: requirePackagePath,
+      };
+    }
+  } catch (err) {
+    console.log(`Cannot resolve ${SDK_PACKAGE_NAME} via require.resolve:`, err);
+  }
+
+  const sdkLocalPackagePath = path.resolve(
+    __dirname,
+    "../../resources/embedding-sdk/dist/main.bundle.js",
+  );
+
+  if (fs.existsSync(sdkLocalPackagePath)) {
+    return {
+      isEmbeddingSdkPackageInstalled: false,
+      embeddingSdkPath: sdkLocalPackagePath,
+    };
+  }
```

Comment:
This is for the E2E workflow (not cross-version), as in the non-cross-version workflow we build the SDK from scratch (which outputs to `resources/embedding-sdk`), and we do not install the package from NPM.